CONTRIBUTING.md

```diff
@@ -1,11 +1,51 @@
-# How to Contribute
+# Contributing to graphql-kimera
+
+> Please note that this project is released with a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
 
 Kimera is one of [Lola Tech's](https://www.lola.tech) first open source project that is both under very active development and is also being used to ship code to our enterprise customers.
 
-## Code of Conduct
+## Found an Issue?
+
+Thank you for reporting any issues you find. We do our best to test and make graphql-kimera as solid as possible, but any reported issue is a real help.
+
+> When raising an issue please use the template provided as a guideline, provided to you.
+
+## Want a new feature?
+
+> For raising a feature request please use the template provided as a guideline, provided to you.
+
+## Branch Organization
+
+Submit all changes directly to the `master` branch. We don’t use separate branches for development or for upcoming releases. We do our best to keep master in good shape, with all tests passing.
```

Comment:
Shouldn't we explain that work should be done on a separate branch and they should target the `master` branch in the PR?

Reply:
Please create a local branch off of `master`, commit your bug report or feature request, then push and create a PR against `master`. We don’t use separate branches for development or for upcoming releases. We do our best to keep master in good shape, with all tests passing.